api/services/SiteCreator.js

```diff
@@ -81,6 +81,7 @@ const createSiteFromExistingRepo = ({ siteParams, user }) => {
 
 const createSiteFromTemplate = ({ siteParams, user, template }) => {
   let site = Site.build(siteParams)
+  site.engine = "jekyll"
```

Comment:
No, then we'd be coupling the site creation code with our assumptions around how templated sites are created. Not good.